std-bits/table/src/main/java/org/enso/table/read/ExcelReader.java

```diff
@@ -430,7 +432,11 @@ private static Table readTable(
             .toArray(Column[]::new);
 
     if (columns.length == 0) {
-      throw new EmptySheetException();
+      var emptySheetType = EnsoMeta.getType("Standard.Table.Errors", "Empty_Sheet");
+      var emptySheetError = emptySheetType.invokeMember("Error");
+      var errorType = EnsoMeta.getType("Standard.Base.Error", "Error");
+      var error = errorType.invokeMember("throw", emptySheetError);
+      throw error.throwException();
```

Comment:
- @jdunkerley, @AdRiley: I'd like to propose this as a new way to throw Enso error from Java code
- construct the right `payload` - e.g. `Empty_Sheet.Error`
- construct _dataflow error_ object by calling `Error.throw`
- and either return the _dataflow error_
- or `throwException()` it and `Panic.recover` [later](https://github.com/enso-org/enso/pull/14040/files#r2517305302) 
- just [simply recover](https://github.com/enso-org/enso/pull/14040/files#r2517305302) - no need for any conversions of `EmptySheetException` Java exception to Enso payload